user_category.py

```diff
@@ -28,26 +28,24 @@ def number():
         else:
             return birth_year
             break
-
 #the determinant value
 user_category_value = current_year - number()
 
 #define function to determine the user category
 def user_type(user_category_value):
+    
     if user_category_value < 18:
-        print ("Your user category is: Minor")
-        return "Your user category is: Minor"
+        cat_message = "Your user category is: Minor"
+        return cat_message
 
     elif user_category_value <= 36:
-        print ("Your user category is: Youth")
-        return "Your user category is: Youth"
+        cat_message = "Your user category is: Youth"
```

Comment:
Why did you choose to create a variable?
I personally find it better to create a variable only if I'm going to use it in many places

Reply:
Alright thanks, I have removed it